packages/event-handler/src/types/rest.ts

```diff
@@ -0,0 +1,24 @@
+import type { GenericLogger } from '@aws-lambda-powertools/commons/types';
+import type { BaseRouter } from '../rest/BaseRouter.js';
+
+/**
+ * Options for the {@link BaseRouter} class
+ */
+type RouterOptions = {
+  /**
+   * A logger instance to be used for logging debug, warning, and error messages.
+   *
+   * When no logger is provided, we'll only log warnings and errors using the global `console` object.
+   */
+  logger?: GenericLogger;
+};
+
+// biome-ignore lint/suspicious/noExplicitAny: we want to keep arguments and return types as any to accept any type of function
+type RouteHandler<T = any, R = any> = (...args: T[]) => R;
+
+type RouteOptions = {
+  method?: string;
+  path?: string;
```

Comment:
Should this type be something like ``/${string}`` to enforce it starting with a forward slash?